packages/components/src/context-tag/context-tag.tsx

```diff
@@ -0,0 +1,38 @@
+import { styled } from '@tamagui/core'
+import { View } from 'react-native'
+
+import { Avatar } from '../avatar'
+import { Text } from '../text'
+
+type Props = {
+  children: React.ReactNode
+  icon?: string
```

Comment:
I would keep this consistent with what `<Avatar />` accepts. At first I thought it accepts an Icon component from `@status-im/icons`.